canu/generate/switch/config/config.py

```diff
@@ -576,6 +577,25 @@ def add_custom_config(custom_config, switch_config, host, switch_os, custom_file
     return custom_config_merge
 
 
+def get_rack_vlans(sls_variables, rack_vlans, destination_rack_list, switch_name):
+    """Get list of VLANs based on rack."""
+    rack_vlans_list = []
+    for cabinets in sls_variables[rack_vlans]:
+        ip_address = netaddr.IPNetwork(cabinets["CIDR"])
```

Comment:
Here you assume that cabinets CIDR in SLS is actually a network address.  That assumes that CSI always does network math correctly.  I'd recommend catching this exception and printing out an error message that leads the user back to SLS data.

Reply:
Alternatively... create another PR where the json load of sls is replaced by the sls_utils work.  sls_utils is in report but not much else and would catch bad network addrs.

Reply:
Created a Jira to add sls_utils to switch config generation.
CASMNET-2184